app/components/Modals/SendModal/SendModal.jsx

```diff
@@ -4,91 +4,43 @@ import React from 'react'
 import BaseModal from '../BaseModal'
 import ReadCode from './ReadCode'
 import ConfirmDetails from './ConfirmDetails'
-
-import parseQRCode from '../../../util/parseQRCode'
+import type { RecipientData } from '../../../util/parseQRCode'
 
 type Props = {
   hideModal: () => any,
-  showErrorNotification: (error: Object) => any,
-  hideNotification: (id: string) => any,
-  pushQRCodeData: (data: Object) => any
-}
-
-type State = {
-  step: string,
-  error: ?string,
-  recipientData: Object
+  pushQRCodeData: (data: Object) => any,
+  getRecipientData: string => any,
+  clearRecipientData: Function,
```

Comment:
very minor comment to avoid using Function but don't need to change it if the rest of the PR is fine we can merge

Reply:
👍🏻e3ff823